wasm/execution/configuration.py

```diff
@@ -268,106 +304,121 @@ def extend_results(self, values: Iterable[TValue]) -> None:
 
     def pop2_u32(self) -> Tuple[numpy.uint32, numpy.uint32]:
-        a, b = self.frame.active_operand_stack.pop2()
+        a, b = self._operand_stack.pop2()
         return cast(numpy.uint32, a), cast(numpy.uint32, b)
 
     def pop3_u32(self) -> Tuple[numpy.uint32, numpy.uint32, numpy.uint32]:
-        a, b, c = self.frame.active_operand_stack.pop3()
+        a, b, c = self._operand_stack.pop3()
         return cast(numpy.uint32, a), cast(numpy.uint32, b), cast(numpy.uint32, c)
 
     #
     # Pop u64
     #
     def pop_u64(self) -> numpy.uint64:
-        return cast(numpy.uint64, self.frame.active_operand_stack.pop())
+        return cast(numpy.uint64, self._operand_stack.pop())
 
     def pop2_u64(self) -> Tuple[numpy.uint64, numpy.uint64]:
-        a, b = self.frame.active_operand_stack.pop2()
+        a, b = self._operand_stack.pop2()
         return cast(numpy.uint64, a), cast(numpy.uint64, b)
 
     def pop3_u64(self) -> Tuple[numpy.uint64, numpy.uint64, numpy.uint64]:
-        a, b, c = self.frame.active_operand_stack.pop3()
+        a, b, c = self._operand_stack.pop3()
         return cast(numpy.uint64, a), cast(numpy.uint64, b), cast(numpy.uint64, c)
 
     #
     # Pop f32
     #
     def pop_f32(self) -> numpy.float32:
-        return cast(numpy.float32, self.frame.active_operand_stack.pop())
+        return cast(numpy.float32, self._operand_stack.pop())
 
     def pop2_f32(self) -> Tuple[numpy.float32, numpy.float32]:
-        a, b = self.frame.active_operand_stack.pop2()
+        a, b = self._operand_stack.pop2()
         return cast(numpy.float32, a), cast(numpy.float32, b)
 
     def pop3_f32(self) -> Tuple[numpy.float32, numpy.float32, numpy.float32]:
-        a, b, c = self.frame.active_operand_stack.pop3()
+        a, b, c = self._operand_stack.pop3()
         return cast(numpy.float32, a), cast(numpy.float32, b), cast(numpy.float32, c)
 
     #
     # Pop f64
     #
     def pop_f64(self) -> numpy.float64:
-        return cast(numpy.float64, self.frame.active_operand_stack.pop())
+        return cast(numpy.float64, self._operand_stack.pop())
 
     def pop2_f64(self) -> Tuple[numpy.float64, numpy.float64]:
-        a, b = self.frame.active_operand_stack.pop2()
+        a, b = self._operand_stack.pop2()
         return cast(numpy.float64, a), cast(numpy.float64, b)
 
     def pop3_f64(self) -> Tuple[numpy.float64, numpy.float64, numpy.float64]:
-        a, b, c = self.frame.active_operand_stack.pop3()
+        a, b, c = self._operand_stack.pop3()
         return cast(numpy.float64, a), cast(numpy.float64, b), cast(numpy.float64, c)
 
     #
     # Frames
     #
-    @property
-    def frame_stack_size(self) -> int:
-        return len(self.frame_stack)
-
     def push_frame(self, frame: Frame) -> None:
-        self.frame_stack.push(frame)
+        if len(self._frame_stack) > 1024:
+            # This is not part of spec, but this is required to pass tests.
+            # Tests pass with limit 10000, maybe more
```

Comment:
Didn't fully grok this comment. The tests would pass if you check `len(self._frame_stack) > 10000`? Then where did `1024` come from?

Is there a place where we can report that the tests don't seem to match the spec?

Reply:
https://github.com/ethereum/py-wasm/issues/116